<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPasswordPage</class>
 <widget class="QWidget" name="SetPasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 30px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
}
QPushButton:disabled
{
	border-radius:5px;
	font: 30px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>当前密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>170</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>170</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>260</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>260</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确认新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_remind">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3个密码输入完毕后请点击右下角［确认］键。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>80</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-radius: 5px;
	font: 20px, &quot;SimHei&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>550</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-radius: 5px;
	font: 20px, &quot;SimHei&quot;;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_currentPwd">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>110</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_newPwd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirmPwd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pwdConfirm">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_confirm2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>新密码输入不一致，请重新输入。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
